weblate/settings_docker.py

```diff
@@ -42,29 +42,33 @@
 
 MANAGERS = ADMINS
 
-DATABASES = {
-    "default": {
-        # Use 'postgresql' or 'mysql'.
-        "ENGINE": "django.db.backends.postgresql",
-        # Database name.
-        "NAME": os.environ["POSTGRES_DATABASE"],
-        # Database user.
-        "USER": os.environ["POSTGRES_USER"],
-        # Name of role to alter to set parameters in PostgreSQL,
-        # use in case role name is different than user used for authentication.
-        "ALTER_ROLE": os.environ.get(
-            "POSTGRES_ALTER_ROLE", os.environ["POSTGRES_USER"]
-        ),
-        # Database password.
-        "PASSWORD": os.environ["POSTGRES_PASSWORD"],
-        # Set to empty string for localhost.
-        "HOST": os.environ["POSTGRES_HOST"],
-        # Set to empty string for default.
-        "PORT": os.environ["POSTGRES_PORT"],
-        # Customizations for databases.
-        "OPTIONS": {"sslmode": os.environ.get("POSTGRES_SSL_MODE", "prefer")},
-    }
-}
+DATABASES = {}
+
+if os.environ["POSTGRES_HOST"]:
+    DATABASES.update(
+    {
+        "default": {
```

Comment:
Make the update simpler (this will need reindeting rest of the block:

```suggestion
    DATABASES["default"] = {
```

Reply:
Done. Thank you!